assets/prototype/domain/eventEditor/containers/queries/currentUser.js

```diff
@@ -0,0 +1,19 @@
+import gql from 'graphql-tag';
+
+export const GET_CURRENT_USER = gql`
+	query GET_CURRENT_USER {
+		viewer {
```

Comment:
I've this based on this discussion: https://spectrum.chat/wpgraphql/general/how-to-properly-make-a-current-user-query-equivalent-of-wp-get-current-user~0f09ef17-6d12-4df0-9a0b-6da8dc875d73?m=MTU2NDE1Nzc3NDE5Mg==

Reply:
![image](https://user-images.githubusercontent.com/1477453/70347328-fd168b80-1868-11ea-810d-5eb2ac8c31a6.png)